pkg/cfn/template/iam_helpers.go

```diff
@@ -2,14 +2,19 @@ package template
 
 // AttachAllowPolicy constructs a role with allow policy for given resources and actions
 func (t *Template) AttachAllowPolicy(name string, refRole *Value, resources interface{}, actions []string) {
+	t.AttachPolicy(name, refRole, MakePolicyDocument(MapOfInterfaces{
+		"Effect":   "Allow",
+		"Resource": resources,
+		"Action":   actions,
+	}))
+}
+
+// AttachPolicy attaches the specified policy document
```

Comment:
nit: move it up?

Reply:
There's no standard convention for ordering dependent functions (like you said, it's a nit).